src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -25,6 +32,28 @@ export default function WebexMeetingControl({type, meetingID}) {
     </Button>
   );
 
+  let select;
+
+  if (options) {
+    select = (
+      !compact
```

Comment:
what happens if compact is true? It should render just the `select`, without the label.